rdtools/degradation.py

```diff
@@ -239,10 +239,11 @@ def degradation_year_on_year(energy_normalized, recenter=True,
     energy_normalized.index.name = 'dt'
 
     # Detect sub-daily data:
-    if min(np.diff(energy_normalized.index.values, n=1)) < \
-            np.timedelta64(23, 'h'):
-        raise ValueError('energy_normalized must not be '
-                         'more frequent than daily')
+    # disabling this check while we experiment with morning/evening agregation
+    # if min(np.diff(energy_normalized.index.values, n=1)) < \
+    #         np.timedelta64(23, 'h'):
+    #     raise ValueError('energy_normalized must not be '
+    #                      'more frequent than daily')
```

Comment:
Assuming we decide that sub-daily aggregation should be allowed, we ought to remove this vestige